src/Microsoft.Extensions.Hosting.Abstractions/Microsoft.Extensions.Hosting.Abstractions.csproj

```diff
@@ -7,6 +7,14 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>hosting</PackageTags>
     <EnableApiCheck>false</EnableApiCheck>
+    <RootNamespace>Microsoft.Extensions.Hosting</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" />
```

Comment:
We should think about using options as well.

Reply:
There's no Options.Abstractions package. Options is pulled in from Extensions.Hosting via Logging.